### PR TITLE
fix windows uwp compile error

### PR DESCRIPTION
**Pull request by Rechi**

### CODE REVIEW

File: src/apimanager.cpp [hunk 1]
```diff
@@ -35,6 +35,8 @@
 
 #include "client.h"
 #include "apimanager.h"
+
+#include <algorithm>
```
Comment (palinek) [inline src/apimanager.cpp]: I have already pushed this into master :)

**Comment by palinek** [comment]
Should I only cherry-pick the a55981f (fix windows uwp compile error) ? Or will you update the PR as a whole?

**Comment by Rechi** [comment]
@palinek updated

**Comment by palinek** [comment]
Is this a known fix and I should merge it immediately or should I wait for appveyor to confirm the build?

**Comment by Rechi** [comment]
Tested this on my windows system, but didn't had access to it earlier today.

**Comment by Rechi** [comment]
CI isn't a hard requirement, but it is set up for most binary addons.

I pinged our pvr guy to have a look at your addon.

**Pull request merged**